src/pages/Welcome/index.js

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
 import { CreateList } from '../../components/create-list';
 import { JoinList } from '../../components/join-list';
-import ContentContainer from '../../components/content-container';
 import Header from '../../components/title';
+import shoppingWelcomeImage from './shopping-welcome-image.svg';
 
 import './style.css';
 
 export const Welcome = () => (
-  <ContentContainer>
+  <main className="welcome-container">
     <Header className="welcome-header">
-      Welcome to your <br />
-      Smart Shopping list!
+      Welcome to your Smart Shopping List!
     </Header>
-    <div className="welcome-form">
-      <CreateList />
-      <p className="welcome-text center">- or -</p>
-      <p className="welcome-text">
-        Join an existing shopping list
-        <br />
-        by entering a three word token.
-      </p>
-      <JoinList />
-    </div>
-  </ContentContainer>
+    <article className="welcome-content">
+      <img
+        className="shoppin-cart-image"
+        src={shoppingWelcomeImage}
+        alt="person shopping with a shopping cart"
+      />
+      <div className="welcome-form">
+        <CreateList />
+        <p className="welcome-text center">- or -</p>
+        <p className="welcome-text">
+          Join an existing shopping list by entering a three word token.
+        </p>
+        <JoinList />
+      </div>
+    </article>
```

Comment:
```suggestion
    <section className="welcome-content">
      <img
        className="shoppin-cart-image"
        src={shoppingWelcomeImage}
        alt="person shopping with a shopping cart"
      />
      <div className="welcome-form">
        <CreateList />
        <p className="welcome-text center">- or -</p>
        <p className="welcome-text">
          Join an existing shopping list by entering a three word token.
        </p>
        <JoinList />
      </div>
    </section>
```
When using `article` tag, [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#usage_notes) doc says it should identify elements with `h` tags

Reply:
Thanks for your advice, I think is better in the way you say, so I fixed the tag 👏